.github/actions/build-docs/action.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Build client documentation'
+description: 'Generates client documentation using godoc'
+runs:
+  using: 'composite'
+  steps:
+    - name: Write html file to docs directory
+      run: |
+        godoc -url pkg/github.com/pinecone-io/go-pinecone/pinecone/ > ../../../docs/index.html
```

Comment:
Why does the `/docs/` folder need to be up 3 levels, does it not create it at the root of the project by default?